File: php/examples/get_websites.php
<?php
require_once 'base_request.php';

// Example of GET /websites

$debug = true;
$website_url   = $_ENV['WEBSITE_URL'];
$request_data = '';
$request_params = ['url' => $website_url];

send_request('GET', '/websites', $request_params, $request_data);

File: csharp/examples/GetWebsites.cs
using System;
namespace Examples.examples
{
    // Example of GET /websites

    public class GetWebsites : BaseRequest
	{
        private string _websiteUrl;

        public GetWebsites()
		{
            dotenv.net.DotEnv.Load();
            _websiteUrl = Environment.GetEnvironmentVariable("WEBSITE_URL");
            Console.WriteLine($"GetWebsites: _websiteUrl={_websiteUrl}");
        }

        public override string SendRequest()
        {
            return GetRequest($"websites?url={_websiteUrl}");
        }
    }
}

